pinot-controller/src/main/java/org/apache/pinot/controller/helix/ControllerRequestUtils.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.controller.helix;
+
+import java.io.IOException;
+import org.apache.commons.httpclient.methods.PostMethod;
+import org.apache.pinot.common.utils.http.HttpUtils;
+import org.apache.pinot.spi.config.table.TableConfig;
+import org.apache.pinot.spi.config.tenant.Tenant;
+import org.apache.pinot.spi.config.tenant.TenantRole;
+import org.apache.pinot.spi.data.Schema;
+import org.apache.pinot.spi.utils.JsonUtils;
+
+
+public class ControllerRequestUtils {
```

Comment:
i dont have a preference, but in general Client should hav some sort of stateful http connection manager maintain inside. where as this Util is pretty much stateless. 

(however, with @Jackie-Jiang 's comment, i think we need to maintain a connection manager just because the SSL security integration needs to be in). so i guess this eventually needs to happen.